content/en/docs/setup/additional-setup/sidecar-injection/index.md

```diff
@@ -11,38 +11,31 @@ aliases:
 
 ## Injection
 
-Each pod in the mesh must be running an Istio compatible sidecar.
+In order to take advantage of all of Istio's features, pods in the mesh must be running an Istio sidecar proxy.
 
 The following sections describe two
 ways of injecting the Istio sidecar into a pod: manually using the `istioctl`
 CLI tool or automatically using the Istio sidecar injector.
 
-Manual injection modifies the controller configuration, e.g. deployment. It
-does this by modifying the pod template spec such that *all* pods for that
-deployment are created with the injected sidecar. Adding/Updating/Removing
-the sidecar requires modifying the entire deployment.
+Manual injection directly modifies configuration, like deployments, and injects the proxy configuration into it.
 
-Automatic injection injects at pod creation time. The controller resource is
-unmodified. Sidecars can be updated selectively by manually deleting a pods or
-systematically with a deployment rolling update.
+Automatic injection injects at pod creation time using an admission controller.
 
-Manual and automatic injection both use the configuration from the
-`istio-sidecar-injector` and `istio` ConfigMaps in the `istio-system`
-namespace.  Manual injection can also optionally load configuration
-from local files.
+Injection occurs by applying a template defined in the ConfigMap `istio-sidecar-injector`.
 
 ### Manual sidecar injection
 
-Inject the sidecar into the deployment using the in-cluster configuration.
+To manually inject a deployment, use `istioctl`:
 
 {{< text bash >}}
 $ istioctl kube-inject -f @samples/sleep/sleep.yaml@ | kubectl apply -f -
 {{< /text >}}
 
-Alternatively, inject using local copies of the configuration.
+By default, this will use in-cluster configuration. Alternatively, injection can be done using local copies of the configuration.
```

Comment:
```suggestion
By default, this will use the in-cluster configuration. Alternatively, injection can be done using local copies of the configuration.
```